install_config/install/host_preparation.adoc

```diff
@@ -245,7 +245,7 @@ Installation] method, you can easily script a complete installation from a
 kickstart or cloud-init setup, change the default configuration file:
 +
 ----
-# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled --insecure-registry 172.30.0.0/16"' \
+# sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled \ --insecure-registry 172.30.0.0/16"\ --signature-verification=false'\
```

Comment:
If the command is one-line in the docs, do you need the \ after `--selinux-enabled and `--insecure-registry 172.30.0.0/16` and at the end? As in:
`sed -i '/OPTIONS=.*/c\OPTIONS="--selinux-enabled  --insecure-registry 172.30.0.0/16" --signature-verification=false'

When a command is presented in the docs using multiple lines, the `\` denotes that the command should be entered all at once and not each line separately. (does that make sense?)

Reply:
Ah, I see what you mean, Michael. I will correct this and ping you to take another look. Thanks!